rules/js.txt

```diff
@@ -2270,6 +2285,16 @@
 <new FontFaceLoadStatus> = <FontFace>.status;
 <new Promise_FontFace_> = <FontFace>.load();
 <new Promise_FontFace_> = <FontFace>.loaded;
+<new sequence_FontFace_> = new Array(<FontFace>);
+
+<new FontSrc p=0.75> = "<cssproperty_font-src_val>"
+<new FontSrc> = "<cssproperty_font-src_val>", <FontSrc>
```

Comment:
This line, in JavaScript code, is going to have identical effect as the previous one, e.g. after assignment `var001 = "a", "b"`, the value of `var001` is always going to be `"a"`. I'm not sure font source in JavaScript can have multiple values. My suggestion would be to remove this line and leave only the previous one (without the probability attribute which is no longer needed). Alternately, you can remove both of these rules and just put `"<cssproperty_font-src_val>"` where there is now `<FontSrc>`.

Reply:
Fixed by using the recursion in `cssproperty_font-src` and including the whole argument in quotes.